### DISCUSSION TITLE
personal  developers use the sp-api sandbox environment

### DISCUSSION THREAD

**Discussion by coffee-developer** [amzn/selling-partner-api-models#4377] [Q&A]
Hi,I am an individual developer. I do not have corporate company information, cannot register an Amazon sales partner account, and cannot obtain sandbox test appId and other configurations.

**Comment by weilinggu** [comment]
Hi @coffee-developer, 
I think you can register on Seller Central without having to provide company information. See [guidance](https://sell.amazon.com/sell/registration-guide). Once you registered a seller central account, you can create developer profile in Seller Central. 
You can also use some of the sandbox to test API endpoints. See [here](https://developer-docs.amazon.com/sp-api/docs/the-selling-partner-api-sandbox)

Thanks,
Weiling
Amazon Selling Partner Services
